proc/api.thrift

```diff
@@ -0,0 +1,40 @@
+service Proc {
+  
+  exception InvalidRequest {
+    1: optional string message
+  }
+
+  exception ScootTooBusy {
```

Comment:
Sure changed it to CanNotScheduleNow, I want the error name to convey that this error is related to time and not request.  A 500 vs a 400 level error in HTTP